l10n_it_fringe_benefit/wizard/import_aci_table.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2013 Alessandro Camilli
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+from openerp import models, fields, api
+
+
+class wizard_of_import_lead(models.TransientModel):
+    _name = "wizard.import.aci.table"
+    _description = "Wizard to import ACI table"
+
+    operation = fields.Selection([
+        ('new', 'New Table'),
+        ('update', 'Update Table'),
+    ], string='Operation', required=True)
+    table_to_update_id = fields.Many2one('fleet.fringe.benefit.version',
+                                         string='ACI table to update')
+    file_to_import = fields.Binary(string='File CSV to import',  required=True)
+    field_separator_csv = fields.Char(string='Field separator for csv',
+                                      required=True, default=",")
+    name = fields.Char(string='Name')
+    date = fields.Date(string='Date')
```

Comment:
string parameter is useless for date and name fields. It will be created by Odoo